resume.tex

```diff
@@ -0,0 +1,123 @@
+
+\input{command.tex}
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% Now for the actual document:
+
+\begin{document}
+
+\input{header.tex}
+
+%%%%%%%%%%%%%%%%%%%%%%%% 
+\begin{ressection}{Employment}
+	\begin{restitledposition}{Nurx / Thirty Madison}{Staff Software Engineer}{April 2022--August 2024}
+		\ressubitem{\textbf{Job Runner} Designed, implemented, and maintained an in-house asynchronous job running
+		solution. Using RabbitMQ, NestJS, and NodeJS/TypeScript, created a new standardized solution for running async
+		tasks at scale across a suite of microservices.}
+		\ressubitem{\textbf{Kafka Topic Isolation} Isolated Kafka topics within staging environments
+		leading to improved stability across dozens of staging environments leading to increased developer efficiency.}
+		\ressubitem{\textbf{Health Metric Standardization} Combining an internal NPM package, Terminus, and Kubernetes
+		probes developed a reliable set of metrics to monitor service health.}
+		\ressubitem{\textbf{Core Services} Supported a microservices architecture with tools and libraries including:
+		RabbitMQ, Kafka, NestJS, GraphQL, Prisma, Postgres, RoR, and several AWS services.}
+		\ressubitem{\textbf{Architecture Review Group} Led an architecture review group for upcoming major technical projects.
+		Participated in intake/scoping, led review discussion, and fascilitated feedback to engineers.}
+	\end{restitledposition}
+	\begin{restitledposition}{Verikai}{Lead Backend Engineer}{November 2021--March 2022}
+		\ressubitem{\textbf{Machine Learning Model Development:} Built the next iteration of the ML workflow responsible
+		for processing census files using NodeJS and Typescript.}
+		\ressubitem{\textbf{AWS Cloud Solutioning} Worked to create AWS solutions to better position the stack for CI/CD
+		using ECS Fargate.}
+	\end{restitledposition}
+	\begin{restitledposition}{Vikus / HealthcareSource / symplr}{Lead Software Engineer}{December 2014--November 2021}
+    \ressubitem{\textbf{Full Stack Web Development:} Developed an applicant tracking system utilizing C\# and EmberJS.
+		Backend tools included RabbitMQ, Redis, Elasicsearch, and AWS. AWS functions used: Lambda, S3,
+		CloudFormation, CloudFront, et al.}
+
+		\ressubitem{\textbf{Integrations:} Wrote several integrations with outside vendors including, but not
+		limited to, payroll vendors, JobBoards (Indeed, ZipRecruiter, Facebook, Google, et al.), Plivo, GitHub,
+		Atlassian, and various WebHooks.}
+
+		\ressubitem{\textbf{Team Lead:} Led the team in a Kanban methodology of issue prioritization. Was a voice
+		to the greater organization to help drive direction and roadmapping.}
+  \end{restitledposition}
+
+  \begin{restitledposition}{devstudio42 LLC}{Co-Founder / Programmer}{April 2014--December 2014}
```

Comment:
about here is where I get bored and stop reading resumes. I'd focus heavily on wordsmithing and perfecting the things above and less on below. I'd probably even suggest moving "Tooling and Language Experience" further up.

Reply:
What about splitting the work history into two parts. Recent 5 years and shorter abbreviated version in a latter section for 5+yrs stuff.

Reply:
since keywords are a thing, I'd do:
- company
- dates (years only)
- title
- tech (languages and systems)